clients/drcachesim/scheduler/scheduler.h

```diff
@@ -361,33 +333,28 @@ template <typename RecordType, typename ReaderType> class scheduler_tmpl_t {
          * #dynamorio::drmemtrace::scheduler_tmpl_t::scheduler_options_t::quantum_unit.
          */
         uint64_t quantum_duration = 10 * 1000 * 1000;
-        /**
-         * Whether and how input stream inter-dependencies should be considered when
-         * preempting input streams.
-         */
-        dependency_flags_t dependency_flags = DEPENDENCY_IGNORE;
         /**
          * If > 0, diagnostic messages are printed to stderr.  Higher values produce
          * more frequent diagnostics.
          */
         int verbosity = 0;
     };
 
-    /** Constructs options for a parallel run-to-completion schedule. */
+    /** Constructs options for a parallel no-inter-input-dependencies schedule. */
     static scheduler_options_t
     make_scheduler_parallel_ops(int verbosity = 0)
```

Comment:
Kind of a nit, but I'm much more used to "ops" being an abbreviation of operations.  I'd find "opts" to be more self-explanatory here (and elsewhere in the code), though it's not entirely clear "options" is worth abbreviating at all.

Reply:
Renamed to the full `_options`.